src/pages/postgraphile/subscriptions.md

```diff
@@ -12,12 +12,18 @@ Subscriptions notify you when an event occurs on the server side.
 
 _This feature requires PostGraphile v4.4.0 or higher._
 
+### Introduction
+
 Pass `--subscriptions` (or `subscriptions: true`) to PostGraphile and we'll
 enhance GraphiQL with subscription capabilities and give your PostGraphile
-server the power of websocket communications; but you'll notice that your
+server the power of websocket communications. This will enable the websocket endpoint.
+
+Additionally, you'll have to add the [`@graphile/pg-pubsub` plugin](https://www.npmjs.com/package/@graphile/pg-pubsub). It's intended that you use this plugin as a provider of realtime data to other plugins which can use it to add subscription fields to your API. For example, this plugin will add the `@pgSubscriptions` directive to easily define your own subscriptions using LISTEN/NOTIFY with makeExtendSchemaPlugin; and adds the --simple-subscriptions feature which, when enabled, adds a simple listen subscription field to your GraphQL API. See below how to enable the plugin for each approach
+
+Finally, you'll notice that your
 schema still only has `query` and `mutation` types. To add subscriptions to
```

Comment:
```suggestion
schema still only has `query` and `mutation` operations. To add subscriptions to
```

Reply:
why `operations`? AFAIK Query and Mutation are GraphQL types (special because they are entry points, but types).

Just curious, I don't have a strong disagreement with this one, any sensible reader will understand both options

Reply:
Sure - fair question! (Edited above to be "operation types" rather than "operations".)

---

`query` and `mutation` are "operation types" defined in the GraphQL spec:

https://graphql.github.io/graphql-spec/June2018/#sec-Language.Operations

These terms do not change.

---

`Query` and `Mutation` are "types" (note the capital letters) and their names can change (in PostGraphile this would be due to inflector plugins). For example in the StarWars API example the `query` operation type is named `Root`:

https://graphql.org/swapi-graphql/

---

I had the option of either capitalising `query/mutation` to `Query/Mutation` or inserting the word `operation` to make the text more accurate. I decided that the `operation` keyword would be more stable, so went that way :+1:

Reply:
Awesome! thanks for the detailed answer